op-node/rollup/driver/state.go

```diff
@@ -388,6 +391,16 @@ func (s *Driver) eventLoop() {
 			}
 		case respCh := <-s.sequencerActive:
 			respCh <- !s.driverConfig.SequencerStopped
+		case respCh := <-s.seLogLevel:
```

Comment:
Only chain/sync/engine changes should be routed through the driver. The log level handler does not have to be synchronized like this, and is unrelated to the driver. Adding a new interface to the admin API to set the log-level through would be better. And the handler should not be changed like this, or every new call ends up creating a handler wrapping the old one, and won't allow the log-level to become more verbose than the previous log level.

Reply:
sorry for delay, have updated based on your comment
but not sure with this part

> and won't allow the log-level to become more verbose than the previous log level

the logger itself doesn't store the state of previous log level so not sure how to compare it